canonical-web-and-design/ubuntu.com
# Add regex pattern for email address form fields

## Done

- There are many sentry errors caused by malformed or garbage email addresses
- I have added `pattern="^[^ ]+@[^ ]+\.[a-z]{2,6}$"` to all `type="email"` input fields to try to catch most of them before the user submits

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Go to any form and test some [valid and invalid emails](https://gist.github.com/cjaoude/fd9910626629b53c4d25) and see if it helps. Note, many do not work, but most are caught
